src/constants/index.ts

```diff
@@ -0,0 +1,9 @@
+export const DEFAULT_DOMAIN = "meet.jit.si";
+
+export const DEFAULT_OPTIONS = {
+  roomName: "DefaultRoomNameExample",
+  width: "100%",
+  height: "700px",
+};
+
+export const JaaS_DOMAIN = "8x8.vc";
```

Comment:
nit: inconsistent naming, I get why, just seems a bit weird to not have it all uppercase.